src/screens/Ticketing/Tickets/RecentTickets/RecentTicketComponent.tsx

```diff
@@ -119,6 +121,29 @@ export const RecentTicketComponent = ({
               paddingHorizontal: theme.spacings.medium,
               paddingVertical: theme.spacings.medium,
             }}
+            accessible={true}
+            accessibilityLabel={`${getReferenceDataName(
+              preassignedFareProduct,
+              language,
+            )}${t(
+              RecentTicketsTexts.a11yPreLabels.transportModes,
+            )} ${returnModeNames({
+              modes: transportModeTexts,
+              joinSymbol: t(RecentTicketsTexts.a11yPreLabels.and),
+            })} ${t(
+              RecentTicketsTexts.a11yPreLabels.travellers,
+            )}: ${userProfilesWithCount
+              .map((u) => '1' + getReferenceDataName(u, language))
+              .join(', ')}
+              ${
+                fromZone === toZone
+                  ? `${t(
+                      RecentTicketsTexts.a11yPreLabels.zones.oneZone,
+                    )} ${fromZone}`
+                  : `${t(
+                      RecentTicketsTexts.a11yPreLabels.zones.multipleZones,
+                    )} ${fromZone}, ${toZone}`
+              }`}
```

Comment:
trekk denne logikken ut av render for å øke lesbarhet

Reply:
will do

Reply:
Skilte ut i egen funksjon, tenker det ble bedre nå